bin/generate_participant.py

```diff
@@ -1 +1,29 @@
-1
+from pathlib import Path
+Path.cwd()
```

Comment:
For scripts, the most important thing is it runs, it should not need lines that print out output for checking etc. That's things like line 2, 4, 5, 6, 13,